src/modules/comments.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+class Comment {
+  constructor(username, comment, item_id) {
+    this.username = username;
+    this.comment = comment;
+    this.item_id = item_id;
+  }
+}
+
```

Comment:
- [Optional] I noticed that comments disappear when you close the popup is closed and reopened. Kindly look into that and fix it, please.